cranelift/frontend/src/ssa.rs

```diff
@@ -124,12 +116,27 @@ impl SSABlockData {
             .expect("the predecessor you are trying to remove is not declared");
         self.predecessors.swap_remove(pred).block
     }
+
+    fn visit(&mut self, epoch: Epoch) -> Option<Block> {
```

Comment:
Can we have a comment here to describe what the semantics of this function are? E.g., when does it return `Some(...)`, and which block is returned? What must `epoch` be?